app/services/project_custom_field_project_mappings/bulk_create_service.rb

```diff
@@ -72,23 +74,25 @@ def validate_contract(service_call, project_ids)
     end
 
     def perform_bulk_create(service_call)
-      ProjectCustomFieldProjectMapping.import(service_call.result, validate: false)
+      ProjectCustomFieldProjectMapping.insert_all(
+        service_call.result.map { |model| model.attributes.slice("project_id", "custom_field_id") }
+      )
 
       service_call
-    rescue StandardError => e
-      service_call.success = false
-      service_call.errors = e.message
```

Comment:
btw, I got rid of this @ba1ash - should put a smile on your face 😄

Reply:
Yep :)